docs/keto/guides/userset-rewrites.mdx

```diff
@@ -1,56 +1,59 @@
 ---
 id: userset-rewrites
-title: Define global permission rules using Ory Permission Language
-sidebar_label: Global rules
+title: Define global permission rules using the Ory Permission Language
+sidebar_label: Using the Ory Permission Language
 ---
 
 import CodeFromRemote from "@theme/CodeFromRemote"
+import BrowserWindow from "@site/src/theme/BrowserWindow"
 const file = (path) => `https://github.com/ory/keto/blob/master/contrib/rewrites-example/${path}`
 
-This guide explains how to configure namespaces and relations using the
-[Ory Permission Language](../reference/ory-permission-language).
+This guide explains how to configure namespaces and relationship rules using the
+[Ory Permission Language](../reference/ory-permission-language) (OPL). You can then run fine-grained checks against Ory Keto,
+which are answered based on a combination of OPL and the concrete high-level relationships stored.
 
 The example describes a file store. Individual files are organized in a folder hierarchy, and can be accessed by individual users
 or groups of users. Using the Ory Permission Language you can specify that if a user has access to a folder, the user also has
 access to all files in that folder.
 
-## Setup and Configuration
+## Ory Network Setup
 
-First, [install Keto](../install.mdx). Next, create a `keto.yaml` file with this basic content:
+The fastest way to get started with Ory Permissions is using [Ory Console](https://console.ory.sh/). You can register for an Ory
+Network account for free.
```

Comment:
Now this sentence here is a bit weird - the title suggests this is all about OPL and then you're saying that the easiest way to start with Ory Permissions is xxx.

Reply:
How's that weird? The OPL is a subset of Ory Permissions, and to use the OPL you need to use Ory Permissions, and the easiest way to start with that that is the Ory Network.

What would you like to read here?